### PR TITLE
Add Hungarian translations

### PR DESCRIPTION
**Pull request by 1ed**

### CODE REVIEW

File: src/Resources/translations/validators.hu.xlf [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+                <source>This value is not a valid mobile number.</source>
+                <target>Hibás mobil szám.</target>
+            </trans-unit>
+            <trans-unit id="4">
+                <source>This value is not a valid pager number.</source>
+                <target>Hibás csipogó szám.</target>
+            </trans-unit>
+            <trans-unit id="5">
+                <source>This value is not a valid personal number.</source>
+                <target>Hibás központi telefonszám (follow me number).</target>
```
Comment (Nek-) [inline src/Resources/translations/validators.hu.xlf:23]: Is it normal there is still some English here?

Comment (1ed) [reply src/Resources/translations/validators.hu.xlf:23]: I tried my best to translate this accurately, but I think there is not exact term for this in Hungarian, so I left the English term too, as a reference to the exact technical meening.

Comment (Nek-) [reply src/Resources/translations/validators.hu.xlf:23]: Ok this sounds great.

**Comment by Nek-** [comment]
Hello @1ed , thank you very much for the translation you provide! 🙌 

Reviewing your work, I figured out that a translation was missing in the default set. I added [it in here](https://github.com/odolbeau/phone-number-bundle/pull/130/files#diff-5cbe316dd053f5f12ba3961fee6a6480c94ba1e8b201811d8defd45c174d58c4R49). I think it would be a plus to have it in your work too, WDYT?

Anyway, I think your work is good enough to be merged, thanks again.

**Review by Nek- — APPROVED** [review]

**Review by maxhelias — APPROVED** [review]
Can you also add the information to CHANGELOG please ?

**Pull request merged**